references/classification/WildFire/torch/README.md

```diff
@@ -0,0 +1,31 @@
+# Train WildFire
+
+We propose here a demonstration of how to use pyro-vision to train a model to detect wildfire outbreaks.
+
+This script allow to train a binary classification model on our WilfFire dataset. You can train here all models availables on Holocron repository:
+https://github.com/frgfm/Holocron/tree/master/holocron/models
+
+You can either follow the procedure below to train the model on your own machine or click on the following collab link to use our training notebook  on powerful and free machine.
```

Comment:
double space after "notebook"
and "on powerful & free machine" --> "on remote servers for free"